mods/ts/rules/nod-infantry.yaml

```diff
@@ -98,7 +98,7 @@ MHIJACK:
 	Buildable:
 		Queue: Infantry
 		BuildPaletteOrder: 60
-		Prerequisites: ~nahand, natech # natech must be natmpl
+		Prerequisites: ~nahand, natmpl
```

Comment:
I mean you commented this line, I changed the cyborg commando as well, but this won't auto-hide, because the line is unrelated which will probably lead to everyone thinking this has unresolved issues.